content/blog/2020-05-04-dvc-3-years-and-1-0-release.md

```diff
@@ -0,0 +1,198 @@
+---
+title: DVC 3 Years Anniversary and 1.0 Pre-release
+date: 2020-05-04
+description: |
+  Today, we’ve got three big things to announce: 🎉 3rd-year anniversary of DVC,
+  🚀 DVC 1.0 pre-release is ready and ⭐ 5000 GitHub starts.
+
+descriptionLong: |
+  Today, we’ve got three big things to announce.
+
+  🎉 3rd-year anniversary of DVC
+
+  🚀 DVC 1.0 pre-release is ready
+
+  ⭐ DVC has reached 5K GitHub starts (coincidently on the same day)
+
+  We are sharing our learnings from this journey and how they affected the new
+  DVC 1.0 release.
+picture: 2020-05-04/5k_stars.png
+pictureComment: 5000 GitHub stars
+author: dmitry_petrov
+commentsUrl: https://discuss.dvc.org/t/dvc-3-years-anniversary-and-1-0-pre-release/374
+tags:
+  - Release
+  - MLOps
+  - DataOps
+  - CI/CD
+---
+
+## 3 years anniversary!
+
+3 years ago on **May 4th, 2017** the first DVC blog post was published
+[Data Version Control beta release: iterative machine learning](https://blog.dataversioncontrol.com/data-version-control-beta-release-iterative-machine-learning-a7faf7c8be67).
+It was the first DVC tutorial and later we made a redirect to the new tutorial
+page to not to confuse users.
+[The first DVC discussion on Reddit](https://www.reddit.com/r/Python/comments/698ian/dvc_data_scientists_collaboration_and_iterative/).
+A few days later it was
+[republished on other sources](https://www.kdnuggets.com/2017/05/data-version-control-iterative-machine-learning.html)
```

Comment:
I don't think showing these old posts makes a very strong point. I would condense it a lot: 

```Three years ago on **May 4th, 2017**, I published the [first blog post about DVC](https://www.kdnuggets.com/2017/05/data-version-control-iterative-machine-learning.html). Until that point, DVC was a private project between myself and two others, my cofounder Ivan and our collaborator, Ruslan. Today, things look very different.```

or some variant of this